modules/test/playwright/tests/commerce/commerce-order-content-web/commerceAdminOrderDetails.spec.ts

```diff
@@ -534,6 +534,8 @@ test('COMMERCE-11888. As a supplier user, I can edit the order details, payments
 		await commerceAdminOrderDetailsPage.orderItemQuantityColumn('3')
 	).toBeVisible();
 
+	await commerceAdminOrderDetailsPage.orderItemFrameCloseButton.click();
+
```

Comment:
if I'm in time remove this empty line, otherwise it's fine :)

Reply:
all the rest of the code has a blank line so I would leave it as is for consistency